cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -59,6 +61,34 @@ define(
                 });
             };
 
+            describe('supported file formats', function() {
+                it('should not show unsupported file format notification for supported files', function() {
+                    var supportedFiles = {
+                        files: [
+                            {name: 'test-1.mp4', size: 0},
+                            {name: 'test-1.mov', size: 0}
+                        ]
+                    };
+                    this.view.$uploadForm.fileupload('add', supportedFiles);
+                    expect(this.view.fileErrorMsg).toBeNull();
+                });
+                it('should show invalid file format notification for unspoorted files', function() {
+                    var unSupportedFiles = {
+                        files: [
+                            {name: 'test-3.txt', size: 0},
+                            {name: 'test-4.png', size: 0}
+                        ]
+                    };
+                    this.view.$uploadForm.fileupload('add', unSupportedFiles);
+                    expect(this.view.fileErrorMsg).toBeDefined();
+                    expect(this.view.fileErrorMsg.options.title).toEqual('Your file could not be uploaded');
+                    expect(this.view.fileErrorMsg.options.message).toEqual(
+                        'test-3.txt is not in a supported file format. Supported file formats are "' +
+                        this.videoSupportedFileFormats.join(', ') + '".'
+                    );
```

Comment:
@mushtaqak Thanks for the sandbox! Can we make a small change to the punctuation in the message? In the 2nd sentence, can we remove the double quotation marks around the supported file extensions, add the periods in front of each extension, and use "and" between the 2 extensions: "Supported file formats are .mp4 and .mov."